packages/cli/tsconfig.json

```diff
@@ -1,12 +1,21 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "module": "commonjs",
+    "declaration": false,
+    "declarationMap": false,
+    "noImplicitAny": false,
+    "removeComments": true,
     "allowSyntheticDefaultImports": true,
-    "types": ["node", "mocha", "chai", "sinon"],
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** ‏ We don't seem to use these, and it is still experimental, I would say remove it

Reply:
Done.